root/components/common-macros.tt

```diff
@@ -1097,3 +1103,9 @@ note-[%- edit.id -%]-[%- loop.count() -%]
     </div>
   </div>
 [%~ END ~%]
+
+[%~ MACRO bugtracker_url(description) BLOCK;
+    'http://tickets.musicbrainz.org/secure/CreateIssueDetails!init.jspa?' _
```

Comment:
Only in this form, JIRA accepts a seeded description. `http://tickets.musicbrainz.org/secure/CreateIssue.jspa` simply ignored everything beyond project ID and issue type.